src/Routes/ArticleComments.jsx

```diff
@@ -0,0 +1,33 @@
+import { getArticleComments } from "../../utils";
+import { Link, Route, Routes } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const ArticleComments = () => {
+  const [comments, setComments] = useState([]);
+  const id = useParams();
+  useEffect(() => {
+    getArticleComments(id.article_id).then((res) => {
+      setComments(res.data.comments);
+    });
+  }, []);
+  return (
```

Comment:
We should have something that tells the users that there are no comments yet for when an article as 0 comments 🙂